wfl/Controllers/RestaurantsController.cs

```diff
@@ -21,6 +21,17 @@ public class RestaurantsController : ApiController
             return Created(restaurant.Id.ToString(), restaurant);
         }
 
+        [Route("{id}")]
+        [HttpPut]
+        public async Task<IHttpActionResult> UpdateNameAsync([FromUri] string id, [FromBody] RestaurantDto restaurant)
```

Comment:
I would actually just name this "Update" instead of UpdateNameAsync.  It doesn't matter too much, but we'll rename it soon enough since we are using the entire RestaurantDto not just a name string.